Production/test/condorSub/dict_RunIIAutumn18MiniAOD_dyjets.py

```diff
@@ -3,5 +3,11 @@
     "samples": [
         ['RunIIAutumn18MiniAOD.DYJetsToLL_M-50_HT-100to200_TuneCP5_PSweights_13TeV-madgraphMLM-pythia8'],
         ['RunIIAutumn18MiniAOD.DYJetsToLL_M-50_TuneCP5_13TeV-madgraphMLM-pythia8_ext1'],
+        ['RunIIAutumn18MiniAOD.DYJetsToLL_M-50_HT-200to400_TuneCP5_PSweights_13TeV-madgraphMLM-pythia8'],
+        ['RunIIAutumn18MiniAOD.DYJetsToLL_M-50_HT-400to600_TuneCP5_PSweights_13TeV-madgraphMLM-pythia8'],
+        ['RunIIAutumn18MiniAOD.DYJetsToLL_M-50_HT-600to800_TuneCP5_PSweights_13TeV-madgraphMLM-pythia8'],
+        ['RunIIAutumn18MiniAOD.DYJetsToLL_M-50_HT-800to1200_TuneCP5_PSweights_13TeV-madgraphMLM-pythia8'],
+        ['RunIIAutumn18MiniAOD.DYJetsToLL_M-50_TuneCP1_13TeV-madgraphMLM-pythia8'],
```

Comment:
do we really want TuneCP1?

Reply:
We certainly do not. This will be fixed in my next commit.